tests/test_not_imported.py

```diff
@@ -8,16 +8,19 @@ def not_imported() -> NotImported:
     return NotImported("Please install foo")
 
 
+@pytest.mark.unit
 def test_get_attr_on_not_installed(not_imported):
```

Comment:
Looks like you could just mark the whole module with the unit test marker.
https://docs.pytest.org/en/7.1.x/example/markers.html#marking-whole-classes-or-modules
```python
import pytest

from great_expectations.compatibility.not_imported import NotImported

# module level markers
pytestmark = [pytest.mark.unit]
```